From f6b6f6bd6020ded3dc6b115536b5382b65786f72 Mon Sep 17 00:00:00 2001
From: Chen Xiao-Long <chillermillerlong@hotmail.com>
Date: Fri, 06 Jan 2012 11:46:32 +0000
Subject: applet: Fix build with automake 1.11.2

pkglib_DATA cannot be used in automake 1.11.2, so use another
name for the files.

https://bugzilla.gnome.org/show_bug.cgi?id=667323
---
(limited to 'applet')

diff --git a/applet/Makefile-lib.am b/applet/Makefile-lib.am
index 4bd795f..7f75083 100644
--- a/applet/Makefile-lib.am
+++ b/applet/Makefile-lib.am
@@ -28,7 +28,8 @@ GnomeBluetoothApplet_1_0_gir_LIBS = libgnome-bluetooth-applet.la
 GnomeBluetoothApplet_1_0_gir_FILES = $(libgnome_bluetooth_applet_la_SOURCES) ../lib/bluetooth-enums.h
 INTROSPECTION_GIRS += GnomeBluetoothApplet-1.0.gir
 
-pkglib_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
+typelibdir = $(pkglibdir)
+typelib_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
 
-CLEANFILES += $(pkglib_DATA)
+CLEANFILES += $(typelib_DATA)
 endif # HAVE_INTROSPECTION
--
cgit v0.9.0.2
